core/experiment-2/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "webcrumbs-experiment-1",
+  "version": "0.0.0",
+  "scripts": {
+    "install:wc": "yarn install && cd packages/admin && yarn install && cd ../plugin1 && yarn install && cd ../plugin2 && yarn install",
+    "reinstall:wc": "rm -rf yarn.lock node_modules && yarn install && cd packages/admin && rm -rf yarn.lock node_modules .next && yarn install && cd ../plugin1 && rm -rf yarn.lock node_modules && yarn install && cd ../plugin2 && rm -rf yarn.lock node_modules && yarn install",
```

Comment:
It might be good to update the install:wc and reinstall:wc too, considering the current folder structure.